coriolisclient/v1/endpoints.py

```diff
@@ -13,7 +13,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from coriolisclient import base
+from coriolisclient import base, exceptions
```

Comment:
Please always write module imports on separate lines.
I'm not 100% if this is part of pep8 or not, but it's something the whole Python community has pretty much agreed on.